tsdb/db_test.go

```diff
@@ -1994,9 +1994,8 @@ func TestBlockRanges(t *testing.T) {
 	app := db.Appender(ctx)
 	lbl := labels.Labels{{Name: "a", Value: "b"}}
 	_, err = app.Add(lbl, firstBlockMaxT-1, rand.Float64())
-	if err == nil {
-		t.Fatalf("appending a sample with a timestamp covered by a previous block shouldn't be possible")
-	}
+	testutil.Assert(t, err != nil,
```

Comment:
We have `testutil.NotOk` for that :) (same comment in other places where you are having assert on `err != nil`.

Reply:
Yup